python/lsst/ts/standardscripts/auxtel/daytime_checkout/atpneumatics_checkout.py

```diff
@@ -124,22 +138,59 @@ async def run(self):
 
         await self.atcs.enable_ataos_corrections()
 
-        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(timeout=5)
-        self.log.info(
-            f"M1 Air pressure with enabled ATAOS corrections is {m1_pressure.pressure:0.0f} Pascals"
+        # Sleep to allow M1 to arrive at commanded pressure
+        await asyncio.sleep(self.ataos_enabled_delay)
+
+        m1_commanded_pressure = (
+            await self.atcs.rem.ataos.evt_m1CorrectionCompleted.aget(
+                timeout=STD_TIMEOUT
+            )
         )
 
+        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(
+            timeout=STD_TIMEOUT
+        )
+
+        # Check that M1 pressure arrived to ATAOS commanded pressure within
+        # tolerance
+        if (
+            m1_commanded_pressure.pressure * (1.0 - self.ataos_pressure_tolerance)
+            < m1_pressure.pressure
+            < m1_commanded_pressure.pressure * (1.0 + self.ataos_pressure_tolerance)
+        ):
+            self.log.info(
+                f"M1 Air pressure with enabled ATOAS corrections is {m1_pressure.pressure:0.0f} Pascals."
+            )
+        else:
+            raise RuntimeError(
+                f"M1 air pressure is {m1_pressure.pressure:0.0f}, outside acceptable range as commanded by "
+                "ATAOS. It needs to be between "
+                f"{m1_commanded_pressure.pressure*(1.-self.ataos_pressure_tolerance)} and "
+                f"{m1_commanded_pressure.pressure*(1.+self.ataos_pressure_tolerance)}."
+            )
+
         await self.checkpoint("Turning off ATAOS corrections")
 
         # Turn off ATAOS correction(s)
         await self.atcs.disable_ataos_corrections()
 
+        # Sleep to allow M1 to settle down to hard points
```

Comment:
So, similar to the case above, I was thinking we would wait for the pressure to reach the desired value and then give it an additional buffer.

Reply:
Same as above, updated error message to be more informative.